db/lastfm_session.py

```diff
@@ -29,7 +36,8 @@ def load(session_key, api_key):
             })
             row = result.fetchone()
             if row:
-                return Session(row)
+                id, userid, sid, api_key, timestamp = row
```

Comment:
Using sqlalchemy, `row` should also be accessible as a dictionary. It would make this line more explicit. What about `return Session(r["id"], r["userid"], ...)`?